README.md

```diff
@@ -63,6 +63,12 @@ You can enable strict mode like below, which will throw the error instead.
 <Latex strict>{textWithSomeBrokenLatex}</Latex>
 ```
 
+### macros
+
```

Comment:
Could you add sth like

> A collection of custom macros. Each macro is a property with a name like \name (written "\\name" in JavaScript) which maps to a string that describes the expansion of the macro.

(from katex docs)

Reply:
Also we could add a link to the persistent macros, make sure users pass the same object to all Latex react components if they want macro persistence.

Reply:
I fixed it.